aries-site/src/components/content/Status.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box, Text } from 'grommet';
+import { Figma, Grommet } from 'grommet-icons';
+import { IconCircle, IconTriangle } from '../icons';
+
+const types = {
+  figma: {
+    name: 'Figma',
+    icon: <Figma color="plain" />,
+  },
+  grommet: {
+    name: 'Grommet',
+    icon: <Grommet color="plain" />,
+  },
+};
+
+const statuses = {
+  'In Progress': {
+    icon: <IconTriangle size="small" color="status-warning" />,
+  },
+  Complete: {
+    icon: <IconCircle size="small" color="status-ok" />,
+  },
+};
+
+const StatusBadge = ({ status, type }) => {
+  return (
+    <Box direction="row">
```

Comment:
can we do `row-responsive` ? 
![Screen Shot 2020-05-07 at 8 41 41 AM](https://user-images.githubusercontent.com/42451602/81308072-9d522580-903e-11ea-827c-786318bc4882.png)

Reply:
Yes, good call.